Resources/Prototypes/Entities/Objects/Consumable/Food/meat.yml

```diff
@@ -160,6 +160,24 @@
     count: 3
     slice: FoodMeatDuckCutlet
 
+- type: entity
+  name: prime-cut corgi meat
+  parent: FoodMeatBase
+  id: FoodMeatCorgi
+  description: The tainted gift of an evil crime. The meat may be delicous, but at what cost?
+  components:
+  - type: Tag
+    tags:
+    - Raw
+  - type: Sprite
+    state: corgi
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        reagents:
+        - ReagentId: Omnizine
+          Quantity: 25
```

Comment:
way too much omnizine, it should also not be omnizine (and probably just bicaridine or smth)

Reply:
I'm guessing it's based on the warm donk pocket, which depending on codebase is either 6 or 15 omnizine.